pkg/apis/performance/v1/performanceprofile_types.go

```diff
@@ -16,6 +16,8 @@ type PerformanceProfileSpec struct {
 	// CPU defines a set of CPU related parameters.
 	CPU *CPU `json:"cpu,omitempty"`
 	// HugePages defines a set of huge pages related parameters.
+	// It possible to define huge pages with the different size, for example you can in the same time
```

Comment:
It *is* possible?

Reply:
Suggestion: 
It is possible to set huge pages with multiple size values at the same time. For example, hugepages can be set with 1G and 2M, both values will be set on the node by the performance-addon-operator.

Reply:
done